Makefile.am

```diff
@@ -99,7 +99,7 @@ $(OSX_APP)/Contents/PkgInfo:
 
 $(OSX_APP)/Contents/Resources/empty.lproj:
 	$(MKDIR_P) $(@D)
-	@touch $@ 
+	@touch $@
```

Comment:
unrelated but okay

Reply:
IDE does it by default, removes trailing spaces. Neat feature.

Reply:
Well it shouldn't! :joy: